### PR TITLE
Show tags in Overview

### PR DESCRIPTION
**Pull request by jtojnar**
Replace pseudo-tag in Overview dialogue with real tags. Adding tags is not actually supported until hamster-lib starts parsing them, though.

### CODE REVIEW

**Review by elbenfreund — APPROVED** [review]

**Review by elbenfreund — DISMISSED** [review]
This PR breaks ``TestFactBox.test__get_tags_widget`` in ``tests.overview.test_widgets``.

**Review by elbenfreund — APPROVED** [review]

**Comment by elbenfreund** [comment]
lgtm. Will merge after rebase

**Comment by elbenfreund** [comment]
This should deal with #76 as well right? Whilst it will not actually be possible to create tags ``hamster-gtk`` has all its cogs in place once ``hamster-lib`` gets to it?!

**Comment by jtojnar** [comment]
Once #150 is merged, we will need it again.

**Pull request merged**
